src/fhirtypes/ElementDefinition.ts

```diff
@@ -1715,6 +1715,7 @@ export class ElementDefinition {
       (type === 'url' && /^\S*$/.test(value)) ||
       (type === 'canonical' && /^\S*$/.test(value)) ||
       (type === 'base64Binary' && /^(\s*([0-9a-zA-Z\+\/=]){4}\s*)+$/.test(value)) ||
+      value.startsWith('ig-loader-') ||
```

Comment:
Due to operator precedence in JS, I think you need to add parentheses around the "or" expression.  E.g.,
`false && false || true` is actually true in JS, so we want something more like `false && (false || true)` to accomplish the boolean logic we're looking for.
```suggestion
      (type === 'base64Binary' && (/^(\s*([0-9a-zA-Z\+\/=]){4}\s*)+$/.test(value)) ||
      value.startsWith('ig-loader-')) ||
```

Reply:
Ooh yes, thank you for catching this!